test/checkpoint/CMakeLists.txt

```diff
@@ -50,14 +50,24 @@ define_test_variants(test_checkpoint
 # Define test variants for the disagg test
 define_test_variants(test_checkpoint
     VARIANTS
-        "disagg_6_row|-d leader -t r -T 6 -x"
+        "disagg_leader_6_row|-d leader -e -t r -T 6 -x"
+        "disagg_leader_6_row_long_running|-d leader -e -t r -W 3 -x -n 1000000 -k 1000000 -C cache_size=4000MB"
```

Comment:
May not be directly related to this PR. I notice all disagg variants has both `-d <disagg_mode>` and `-e` specified. Could the test runner implicitly set `-e` when `-d <disagg_mode>` is specified?